eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -122,9 +122,12 @@ jobs:
 
       - script: |
           pod lib lint --quick
-        displayName: Lint PodSpecs 
+          pod spec lint --use-modular-headers
+        displayName: Lint PodSpecs
 
-      - template: /eng/pipelines/templates/steps/verify-versions.yml
```

Comment:
Probably worth just updating the verify-versions.yml file instead of removing it.

Reply:
We didn't have any other step templates so I tried to match the format in `archetype-sdk-client.yml` which has a bunch of small `script` steps.

Reply:
Given this is likely the only place then doing it directly as a script step makes sense.